pom.xml

```diff
@@ -74,36 +73,42 @@
       <dependency>
         <groupId>org.apache.openwebbeans</groupId>
         <artifactId>openwebbeans-se</artifactId>
-        <version>2.0.5</version>
+        <version>2.0.27</version>
         <scope>test</scope>
       </dependency>
+
+      <dependency>
```

Comment:
is this module needed or can we skip it (even if it costs a small utility class), didnt notice code requiring it

Reply:
For me yes, it is needed.
It was part of play library before but it was [removed in play 2.9](https://www.playframework.com/documentation/3.0.x/Migration29#Removed-libraries); that's why I'm adding it manually now.

It is required at two places to convert Java optional to scala Option :
https://github.com/rmannibucau/playx/blob/f1665edcd1c271a5828d179a5086fb3c0939082c/demo/demo-jaxrs/src/main/java/com/github/rmannibucau/playx/demo/jaxrs/Launch.java#L44

Reply:
Well, IMHO we can drop it, used it cause it was there but the mapping can be quite trivial and avoids this dep which generally ends not very well from my experience, wdyt?

Reply:
All right, I provided a straightforward replacement.